internal/providers/terraform/aws/testdata/global_accelerator_test/global_accelerator_test.golden

```diff
@@ -0,0 +1,10 @@
+
+ Name          Monthly Qty  Unit   Monthly Cost 
+                                                
+ Example                                        
```

Comment:
This should be the addres of the resource. Probably the address just needs to be passed through in the BuildResource function.
```suggestion
 aws_globalaccelerator_accelerator.example                                        
```